### DISCUSSION TITLE
Roadmap / Plans for Bootstap 5

### DISCUSSION THREAD

**Discussion by JiCiT** [snapappointments/bootstrap-select#2580] [General]
Is there a roadmap or any other plans for what's in the works, especially regarding supporting bootstrap 5?  Given that bootstrap has moved away from jQuery to vanilla JS, are there any plans to do the same with bootstrap-select?

**Comment by duhnunes** [comment]
Yeah, as he says in https://github.com/snapappointments/bootstrap-select/issues/2228#issuecomment-711029096 : 

>  I've been absolutely swamped with work as of late, but I'm hoping to get back into continuing maintenance of bootstrap-select very soon. My priorities are:
> 
> 1. Finalize #899. Or, at the very least get it to the point where that feature is in "beta" and previous functionality is not broken.
> 2. Begin immediate work on dropping jQuery requirement.
> 3. Begin work on Bootstrap 5 support.

**Reply by JiCiT** [reply]
Wow pinned right to the top of the issues list even.  Sorry.  Don't know how I missed that before.

**Comment by caseyjhol** [comment]
Support for Bootstrap 5 was added in [v1.14.0-beta2](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.14.0-beta2)! jQuery is still required. You must use Bootstrap v5.0.0-beta3 or later. We have plans to remove the jQuery dependency in v2.0.0.
